docs/docs/integrations/toolkits/polygon.ipynb

```diff
@@ -0,0 +1,85 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e6fd05db-21c2-4227-9900-0840bc62cb31",
+   "metadata": {},
+   "source": [
+    "# Polygon IO Toolkit\n",
+    "\n",
+    "This notebook shows how to use agents to interact with the Polygon IO toolkit. The toolkit provides access to Polygon's Stock Market Data API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d93e6bd-03d7-4d3c-b915-8b73164e2ad8",
+   "metadata": {},
+   "source": [
+    "## Example Use:\n",
+    "---\n",
+    "### Initializing the agent"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "648a2cb2-308e-4b2e-9b73-37109be4e258",
+   "metadata": {
+    "is_executing": true
+   },
+   "outputs": [],
+   "source": [
+    "from langchain.agents import AgentType, initialize_agent\n",
+    "from langchain_community.agent_toolkits.polygon.toolkit import PolygonToolkit\n",
+    "from langchain_community.utilities.polygon import PolygonAPIWrapper\n",
+    "from langchain_openai import OpenAI\n",
+    "\n",
+    "llm = OpenAI(temperature=0, openai_api_key=\"\")\n",
+    "polygon = PolygonAPIWrapper()\n",
+    "toolkit = PolygonToolkit.from_polygon_api_wrapper(polygon)\n",
+    "agent = initialize_agent(\n",
```

Comment:
can we not use initialize agent, but rather match the notebook here: https://python.langchain.com/docs/integrations/toolkits/gmail